src/main/java/frc/team3128/commands/CmdAlign.java

```diff
@@ -70,16 +74,16 @@ public void execute() {
                 if(!limelights.getShooterHasValidTarget()) {
                     aimState = VisionState.SEARCHING;
                     plateauCount = 0;
+                    controller.enableContinuousInput(-180, 180);
                     break;
                 }
 
                 // turn with PID loop using input as horizontal tx error to target
                 currHorizontalOffset = limelights.getShooterTX();
                 currError = goalHorizontalOffset - currHorizontalOffset; // currError is positive if we are too far left
-
-                double ff = Math.signum(currError) * VISION_PID_kF;
-                double feedbackPower = VISION_PID_kP * currError + VISION_PID_kD * (currError - prevError) / (currTime - prevTime) + ff;
                 
+                double feedbackPower = controller.calculate(currHorizontalOffset);
```

Comment:
keep the ff line in here and set feedbackPower = the code you have in here with .calculate() + ff
`double ff = Math.signum(currError) * VISION_PID_kF`